mlem/contrib/kubernetes/context.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os
+from enum import Enum
+from typing import ClassVar
+
+from pydantic import BaseModel
+
+from mlem.utils.templates import TemplateModel
+
+logger = logging.getLogger(__name__)
+
+
+class ImagePullPolicy(str, Enum):
+    always = "Always"
+    never = "Never"
+    if_not_present = "IfNotPresent"
+
+
+class ServiceTypeEnum(str, Enum):
```

Comment:
Do you still need this?

Reply:
hmm, not really, but it gives a signal that there are limited types available
and of course will error out if we supply an invalid value.
So I am slightly inclined to keep it